docs/platform/howto/manage-vpc-peering.rst

```diff
@@ -81,9 +81,15 @@ IP filtering (the **Allowed IP addresses** list on the service's **Overview** pa
 Also note that safelisting applies to both internal and external traffic. If you safelist an external IP address and want to keep traffic flowing with the internal (peered) connections, make sure that you safelist the CIDR blocks of the peered networks as well to avoid disruptions to the service.
 
 Troubleshoot VPC connection issues
--------------------------------------
+----------------------------------
 
-Any network changes to VPC peered hosts external from Aiven can cause issues with routing to your Aiven services hosted in a VPC. To troubleshoot such issues, take the following steps:
+Any network changes to VPC peered hosts external from Aiven can cause issues with routing to your Aiven services hosted in a VPC. To troubleshoot such issues, :ref:`refresh VPC connections <refresh-vcp-connections>`.
+
+Changes to your VPCs (such as adding a new subent) can take up to 24 hours to take effect. There's no need to troubleshoot issues with your changes not applied unless it's been over 24 hours since you introduced the changes. You can always force immediate  application of your changes by :ref:`refreshing VPC connections <refresh-vcp-connections>`.
```

Comment:
```suggestion
Changes to your VPCs (such as adding a new subnet) can take up to 24 hours to take effect. There's no need to troubleshoot issues with your changes not applied unless it's been over 24 hours since you introduced the changes. You can always force immediate  application of your changes by :ref:`refreshing VPC connections <refresh-vcp-connections>`.
```